docs/_includes/footer.html

```diff
@@ -36,7 +36,7 @@
   <div class="footer-icons-list">
 
     <div class="footer-icons">
-      <a class="social__link social__link--mail" href="mailto:community@layer5.io" width="50px" height="50px">
+      <a class="social__link social__link--mail" href="mailto:community@layer5.io">
```

Comment:
@raghavaggarwal99 this dint seem to fix the issue because the icons have completely disappeared from the screen when I tested it. @leecalcote reported the same behaviour.

Reply:
May be there is a cache issue on your end. Can you please clear your browser cache and try once?

Reply:
Yes so the issue is so it's showing correctly in firefox private window but not in google chrome

Reply:
@raghavaggarwal99 thanks for working through this. Be sure to also note the positioning of the footer, which should be centered on the page.